vllm/sampling_params.py

```diff
@@ -237,6 +238,16 @@ def sampling_type(self) -> SamplingType:
             return SamplingType.RANDOM_SEED
         return SamplingType.RANDOM
 
+    def clone(self) -> "SamplingParams":
+        """Deep copy excluding LogitsProcessor objects, which may contain an
+            arbitrary/nontrivial amount of data."""
+
```

Comment:
in a comment here, could we link to the issue as well?

Reply:
Now added!